invenio_rdm_records/services/services.py

```diff
@@ -21,6 +21,36 @@ class RDMRecordService(RecordDraftService):
     config_name = "RDM_RECORDS_BIBLIOGRAPHIC_SERVICE_CONFIG"
     default_config = config.RDMRecordServiceConfig
 
+    def rebuild_index(self, identity):
```

Comment:
Will you move this method to drafts-resources similar to your comment in invenio-vocabularies so we only keep the cli command above here?